packages/destination-actions/src/destinations/hubspot/sendCustomBehavioralEvent/index.ts

```diff
@@ -77,6 +77,14 @@ const action: ActionDefinition<Settings, Payload> = {
       properties: flattenObject(payload.properties)
     }
 
+    const hubId = settings?.portalId
+
+    if (!hubId && !/^pe\d+_.*/.exec(payload?.eventName)) {
+      throw new PayloadValidationError(`EventName should begin with pe<hubId>_`)
+    } else if (hubId && !payload?.eventName.startsWith(`pe${hubId}_`)) {
+      throw new PayloadValidationError(`EventName should begin with pe${hubId}_`)
+    }
```

Comment:
Since we are returning early, else is not needed. Could you extract the regex to a variable for readability?
```suggestion
    } 
    if (hubId && !payload?.eventName.startsWith(`pe${hubId}_`)) {
      throw new PayloadValidationError(`EventName should begin with pe${hubId}_`)
    }
```